test/twofishes/client_test.rb

```diff
@@ -3,35 +3,81 @@
 module Twofishes
   class Zurich
     def self.geocode(location)
-      YAML::load_file('test/fixtures/response.yaml')
+      if location.responseIncludes == [ResponseIncludes::PARENTS]
```

Comment:
This feels really weird, especially now that it has so much logic in it. Maybe I should re-think the way we test this.